### PR TITLE
Release v9.125.0

### PR DESCRIPTION
**Pull request by emersonlaurentino**
#### What problem is this solving?
[Running workspace](http://link)

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [9.125.0] - 2020-07-23
+
```
Comment (estacioneto) [inline CHANGELOG.md:11]: Nice! ✨ 
Just a comment. Should this _Hide clear button on `InputSearch` when disabled`_ entry be on `### Fixed` section?

Comment (emersonlaurentino) [reply CHANGELOG.md:11]: Idk, I just rebase, it was already like that in PR.

Comment (estacioneto) [reply CHANGELOG.md:11]: @vlaux what do you think?

Comment (estacioneto) [reply CHANGELOG.md:11]: Nevermind... Already merged 🤷

**Pull request merged**